featureFlags/README.md

```diff
@@ -0,0 +1,150 @@
+# FeatureFlags
+
+# FeatureFlag object
+
+```
+{
+    'name': string,
+    'title':string,
+    'id': string,
+    'created_at': timestamp,
+    'updated_at': timestamp,
+    'config': {'enabled': boolean},
+    'owner': string,
+    'launched_at': timestamp
+}
+```
+
+## **Requests**
+
+|                 Route                 |             Description                        |
+|:-------------------------------------:|:----------------------------------------------:|
+|  [GET /featureFlags](#get-featureFlags)        |   Returns all feature flags in system |
+|  [POST /featureFlags](#post-featureFlags)      |   Creates a feature flag              |
+|  [PATCH /featureFlags/:id](#patch-featureFlagsId)   | Updates data of the feature flag |
+|  [DELETE /featureFlags/:id](#delete-featureFlagsId) |   Deletes the feature flag       |
+
+
+## **GET /featureFlags**
+
+ Returns all feature flags in the system 
+
+- **Params**
+   None
+- **Query**
+   None
+- **Body**
+   None
+- **Headers**
+   None
+- **Cookie**
+   None    
+- **Success Response:**
+- **Code:** 200
+  - **Content:**
+
+```
+{
+  message: 'FeatureFlags returned successfully!'
+  featureFlags: [
+           {<featureFlag_object>},
+           {<featureFlag_object>}
+         ]
+}
+```
+
+- **Error Response:**
+  - **Code:** 500
+    - **Content:** `{ 'statusCode': 500, 'error': 'Internal Server Error', 'message': 'An internal server error occurred' }`
+
+## **POST /featureFlags** 
+
+Creates a feature flag 
+
+- **Params**  
+  None
+- **Query**  
+  None
+- **Headers**  
+  Content-Type: application/json
+- **Cookie**  
+  rds-session: `<JWT>`
+- **Body** `
+{ 
+   _Required:_ 'feature_name'= <feature name>,
+   _Required:_ 'feature_owner'= <feature owner name>,
+   _Required:_ 'is_enabled'= <feature is enabled or disabled>,             
+ }`
+- **Success Response:**
+- **Code:** 200
+  - **Content:**
+
+```
+{
+  message: 'FeatureFlag created successfully!'
+  featureFlag: {<featureFlag_object>}
+  id: <newly created featureFlag id>
```

Comment:
The `id` would be part of the `featureFlag` object itself. I don't think a separate field is needed.